Landing page uses react-boostrap components and props

ejuor opened the pull request:
Hey, @nionata @hoatran1403 there's still a bug in this code where smooth scroll doesn't work in IE (not tested in Safari). Can you guys check and maybe point out what I did wrong?

This resolves #9.

hoatran1403 commented:
I've tested with safari 10.0.1 and it worked well. i'm trying to find a way to install IE on mac for testing.

ejuor commented:
I fixed it @hoatran1403 but I added some code to @nionata's smooth scroll. Basically (on `./src/components/landingpage/smoothscroll.js`), when the `if` conditional expression calls `this.hostname` it would always return an empty string (on Internet Explorer), and comparing that string to `location.hostname` will evaluate to `false` so the whole block inside the `if` statement wouldn't run everytime. 😫
